fabric-renderer-api-v1/src/client/java/net/fabricmc/fabric/api/renderer/v1/material/MaterialFinder.java

```diff
@@ -34,45 +35,56 @@
  */
 public interface MaterialFinder extends MaterialView {
 	/**
-	 * Defines how sprite pixels will be blended with the scene.
+	 * Controls how sprite pixels should be blended with the scene.
 	 *
-	 * <p>See {@link BlendMode} for more information.
+	 * <p>The default value is {@link BlendMode#DEFAULT}.
+	 *
+	 * @see BlendMode
 	 */
 	MaterialFinder blendMode(BlendMode blendMode);
 
 	/**
-	 * Vertex color(s) will be modified for quad color index unless disabled.
+	 * Controls whether vertex colors should be modified for quad coloring.
+	 *
+	 * <p>The default value is {@code false}.
```

Comment:
That's not very clear. The default value is `{@code false}`, which means that the vertex colors _will be modified_. What about this?

```suggestion
	 * Controls whether vertex colors should be modified for quad coloring.
	 *
	 * <p>The default value is {@code false}, meaning that vertices will be colored according to the color index.
```